Modules/Module_Processing/Module_ASL/xASL_wrp_RealignASL.m

```diff
@@ -526,7 +526,7 @@ function xASL_wrp_RealignASL(x, bASL)
 			jsonFields.Q.LabelingDuration = x.Q.LabelingDuration(~exclusion);
 		end
 
-        xASL_io_SaveNifti(x.P.Path_ASL4D, x.P.Path_despiked_ASL4D, NewIm, 32, 0, [], 1, jsonFields, true);
+        xASL_io_SaveNifti(x.P.Path_ASL4D, x.P.Path_despiked_ASL4D, NewIm, 32, 0, [], 1, jsonFields, true, [true true, false]);

```

Comment:
VERY GOOD POINT!! Shall I check for all other newly added SaveNiftis? Maybe we have messed up this on more occassions.

Reply:
We can check this, definitely. I guess that we do this correctly on other locations, because you always want to have the JSON & .mat of the new NIfTI. Here specifically we recreate the .mat ourselves.

One other way to do this, is that if we recreate JSON or .mat "manually" (i.e., outside of xASL_io_SaveNifti), it is safer to do this after xASL_io_SaveNifti rather than before.

Reply:
OK. I guess we can resolve then